third_party/docfx/templates/devsite/layout/_master.tmpl

```diff
@@ -5,18 +5,10 @@
 {{!include(logo.svg)}}
 {{!include(search-stopwords.json)}}
 <!DOCTYPE html>
-<!--[if IE]><![endif]-->
 <html devsite>
   {{>partials/head}}
   <body data-spy="scroll" data-target="#affix" data-offset="120">
-    <div id="wrapper">
       <header>
-        {{^_disableNavbar}}
-        {{>partials/navbar}}
-        {{/_disableNavbar}}
-        {{^_disableBreadcrumb}}
-        {{>partials/breadcrumb}}
-        {{/_disableBreadcrumb}}
       </header>
       {{#_enableSearch}}
       <div class="container body-content">

```

Comment:
I think we can get rid of a couple other things below -- don't think we need affix or toc. Or the `col-md-*` classes.

Reply:
done, got rid of `affix`, `toc`, `col-md-*` divs and adjusted alignment accordingly